caleb-lab/test/test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+//require chai, buffer [node], and the filesystem
+const expect = require('chai').expect;
+const fs = require('fs');
+
+//describe the full module
+describe('fs module', function(){
+  //describe what we're trying to do with the things in the module
+  describe('#readFile', function(){
+    //describe what we expect with the test
+    it('should read a file and produce a raw buffer', done => {
+      fs.readFile(`${__dirname}/../data/text.txt`, function(err, data){
+        let bufferData = data;
+
+        //we expect this to be the data that gets operated on in the index.js, and for it to be a type of buffer. I intend to ask for more clarification on the "from". I particularly hate writing things that I do not understand.
+        expect(typeof bufferData).to.equal(typeof Buffer.from(''));
```

Comment:
https://nodejs.org/api/buffer.html#buffer_class_method_buffer_from_array

Basically, ``Buffer.from('')`` creates a new buffer array, it's the same as just saying something like:

```
let newArray = []'

```